kahuna/public/js/components/gr-delete-image/gr-delete-image.js

```diff
@@ -10,12 +10,12 @@ deleteImage.controller('grDeleteImageCtrl', ['mediaApi', function (mediaApi) {
         mediaApi.delete(ctrl.image)
             .then((resp) => {
                 if (angular.isDefined(ctrl.onSuccess)) {
-                    ctrl.onSuccess(resp);
+                    ctrl.onSuccess(resp, ctrl.image);
```

Comment:
need the image so we can do this https://github.com/guardian/grid/pull/1076/files#diff-8357e32a30a84e3c0778b7128dc09891R39